src/core/blockchain/zilliqa/transaction.ts

```diff
@@ -86,7 +86,7 @@ export const buildTransferTransaction = async (
 
                 toAddress: tokenInfo.contractAddress,
 
-                amount: '0',
+                amount: tx.amount,
```

Comment:
this is not ok, amount should remain `0` for ZRC2 transactions

Reply:
I used `tx.amount` in order to fix the amount on transaction history. 
So because amount should remain `0` for ZRC2 transactions, how to fix the amount on transaction history?